autoware_launch/rviz/autoware.rviz

```diff
@@ -718,7 +718,7 @@ Visualization Manager:
               Name: DynamicObjects
               Namespaces:
                 label: true
-                path: false
+                path: true
```

Comment:
Path flag is false for tracking.
https://github.com/tier4/autoware_launcher/blob/cdc3e2de5882445ad8d468a1788c95ed508a71c0/autoware_launch/rviz/autoware.rviz#L616-L621

Reply:
Fixed at 0da8afb .